src/board/board.py

```diff
@@ -1,98 +1,126 @@
-from src import config
+import copy
+import logging
+
 from src.colors import colors
 from src.point.point import Point
 from src.randomizer.randomizer import Randomizer
 from src.renderer.renderer import Renderer
+from src.tetromino.constants import COLORS, SPAWN
 from src.tetromino.tetromino import Tetromino
 
+log = logging.getLogger(__name__)
+
 
 class Board:
     """Board contains all the tetrominos in the current game"""
 
     def __init__(self, width, height):
+        log.info("Initializing board (width={}, height={})".format(width, height))
         self.width = width
         self.height = height
-        self.board_matrix = [[0 for y in range(height)] for x in range(width)]
-        self.piece_matrix = [[0 for y in range(height)] for x in range(width)]
         self.random_list = Randomizer()
-        next_piece = self.random_list.next()
+        self.next_tetromino_name = self.random_list.next()
```

Comment:
Hmm I feel next_tetromino sounds better what do you think?

Reply:
I did it this way because random_list.next() returns a string, not a tetromino object. 
Now that I think about it, maybe returning a tetromino object would be better :open_mouth:

Reply:
I think you should pass self.tetromino.name

Reply:
Like it was before? How so?